modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java

```diff
@@ -284,9 +304,51 @@ private void _validate(long objectViewId, long objectDefinitionId)
 		}
 	}
 
+	private void _validateSortColumn(
+			List<ObjectViewSortColumn> objectViewSortColumns,
+			ObjectView objectView)
+		throws ObjectViewSortColumnException {
+
+		List<ObjectField> objectFields =
+			_objectFieldPersistence.findByObjectDefinitionId(
+				objectView.getObjectDefinitionId());
+
+		List<String> objectFieldLabel = new ArrayList<>();
+
+		for (ObjectField objectField : objectFields) {
+			objectFieldLabel.add(objectField.getName());
+		}
+
+		List<String> sortOrders = new ArrayList<String>() {
```

Comment:
I'm afraid I have to disagree with creating this variable inside this method. Could you read the comment below?